src/EventStore.TestClient/Commands/DvuBasic/DvuBasicProcessor.cs

```diff
@@ -296,6 +293,7 @@ private void Write(Status status, int writerIdx, CommandProcessorContext context
                                         wrongExpectedVersion, streamsDeleted, failed, requests);
             status.FinilizeStatus(writerIdx, failed != sent);
             connection.Close();
+            finish.Set();
```

Comment:
With `finish.Set()` called in the packageHandler callback, the last writer to finish would sometimes miss calling the `status.FinilizeStatus` method before results were tallied